test/correctness/loop_carry.cpp

```diff
@@ -0,0 +1,178 @@
+#include "Halide.h"
+
+#include "../performance/benchmark.h"
+
+using namespace Halide;
+
+void dump_asm(Func f) {
+    Target t("host-no_runtime-no_asserts-no_bounds_query");
+    f.compile_to_assembly("/dev/stdout", {}, t);
+}
+
+int main(int argc, char **argv) {
```

Comment:
Does any of this even test loop carry? There's no mention of hexagon here, and loop carry is only turned on for hexagon...